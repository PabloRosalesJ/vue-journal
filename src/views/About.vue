<template>
  <div class="about">
    <h1>This is an about page :D</h1>
  </div>
</template>
