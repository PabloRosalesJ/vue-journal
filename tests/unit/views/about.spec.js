import { shallowMount } from '@vue/test-utils'
import About from '@/views/About'

describe('Pruebas en el About view', function () {

  test('Debe renderizarse el componente correctamente', () => {
    
    const wraper = shallowMount( About )
    expect( wraper.html() ).toMatchSnapshot()

  })
  
})