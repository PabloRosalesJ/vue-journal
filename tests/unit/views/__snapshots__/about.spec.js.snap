// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el About view Debe renderizarse el componente correctamente 1`] = `
<div class="about">
  <h1>This is an about page :D</h1>
</div>
`;
